dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -368,8 +370,9 @@ public void doImport(PrintWriter out) throws IOException {
         }
 
         Collections.sort(roles);
-        try{
-            HibernateUtil.closeSession();
+
+        try {
+            HibernateUtil.closeSession();   // todo: why this?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)